### PR TITLE
Make komodo py3.12 compatible

### PR DESCRIPTION
**Pull request by jonathan-eq**

### CODE REVIEW

**Comment by jonathan-eq** [comment]
So the integration test failed for python3.12 because there is no numpy version with support for both py3.8 and py3.12..

**Comment by jonathan-eq** [comment]
Will squash before merge

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -5,13 +5,15 @@ on: [pull_request]
 jobs:
   build:
     env:
-      RELEASE: release-py3x.yml
-
+      RELEASE_3_X: release-py3x.yml
+      REPOSITORY_3_X: repository-py3x.yml
+      RELEASE_3_12: release-py3_12.yml
+      REPOSITORY_3_12: repository-py3_12.yml
```
Comment (berland) [inline .github/workflows/test.yml:11]: numpy just serves as an example. You can probably pick any other package instead, and then keep it simpler.

Comment (jonathan-eq) [reply .github/workflows/test.yml]: Yes, I overcomplicated it a little bit

**Review by berland — APPROVED** [review]
Nice, remember to clean up commit history.

**Pull request merged**